src/components/LoadingModal/LoadingModal.tsx

```diff
@@ -29,7 +29,7 @@ const useStyles = makeStyles(theme => ({
 export const LoadingModal = (props: {
   visible: boolean;
   onClose?: () => void;
-  text: string;
+  text?: string;
 }) => {
   const classes = useStyles();
```

Comment:
Sorry, possible candidate for a stitches rewrite? If it's not too much trouble ofc. Entirely up to you.

Reply:
I'm going to hold off on this. The circular progress bar is a Material-UI component